test/unit/tasks/cms/cms_test.rb

```diff
@@ -22,4 +22,31 @@ def test_fix_pagination_href
       assert_equal false, @page.published.include?('{{ part.link }}')
     end
   end
+
+  class FixInvalidSysNameSections < ActiveSupport::TestCase
+    def setup
+      @provider = FactoryBot.create(:provider_account)
+    end
+
+    test 'fixes invalid system names' do
+      section = FactoryBot.build(:cms_section, partial_path: '/', parent: @provider.sections.root, title: 'New section',
+                                 system_name: 'invalid name')
+      section.save!(validate: false)
+      expected_sysname = "section-#{section.id}"
+
+      execute_rake_task 'cms/cms.rake', 'cms:fix:section_invalid_system_names'
+
+      assert_equal expected_sysname, section.reload.system_name
+    end
+
+    test "doesn't change valid system names" do
+      expected_sysname = "new-section"
```

Comment:
We should use single quotes when there's no string interpolation.

Reply:
Commited